subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -1190,6 +1200,116 @@ static uint16_t l2cap_br_conf_opt_mtu(struct bt_l2cap_chan *chan,
 	return result;
 }
 
+static uint16_t l2cap_br_conf_opt_flush_timeout(struct bt_l2cap_chan *chan,
+				   struct net_buf *buf, size_t len)
+{
+	uint16_t result = BT_L2CAP_CONF_SUCCESS;
+	struct bt_l2cap_conf_opt_flush_timeout *opt_to;
+
+	if (len != sizeof(*opt_to)) {
+		LOG_ERR("qos frame length %zu invalid", len);
+		result = BT_L2CAP_CONF_REJECT;
+		goto done;
+	}
+
+	opt_to = (struct bt_l2cap_conf_opt_flush_timeout *)buf->data;
+
+	LOG_DBG("Flash timeout %u", opt_to->timeout);
+
+	opt_to->timeout = sys_cpu_to_le16(0xFFFF);
+	result = BT_L2CAP_CONF_UNACCEPT;
+done:
+	return result;
+}
+
+static uint16_t l2cap_br_conf_opt_qos(struct bt_l2cap_chan *chan,
+				   struct net_buf *buf, size_t len)
```

Comment:
Looks like incorrect alignment of the split line (again) here. It would be good if you could try to configure your editor so that it just does the right thing and you don't have to think about this :)

Reply:
Updated.